### DISCUSSION TITLE
Crazyflie Constraints

### DISCUSSION THREAD

**Discussion by OmarShadeed** [bitcraze/discussions#37] [Q&A]
Hi, 
I just wanted to ask if anyone performed tests to know the crazyflie dynamical limits. I mean in terms of maximum speed and acceleration. 
Thanks,

**Comment by knmcguire** [comment]
Not sure if this is shown, but you can check out this thesis: [System Identification of the Crazyflie 2.0 Nano Quadrocopter - Research Collection (ethz.ch)](https://www.research-collection.ethz.ch/handle/20.500.11850/214143)
